relayer/go.mod

```diff
@@ -1,86 +1,96 @@
 module github.com/smartcontractkit/chainlink-starknet/relayer
 
-go 1.21
+go 1.21.3
 
-toolchain go1.21.1
+toolchain go1.21.5
 
 require (
-	github.com/ethereum/go-ethereum v1.11.5
+	github.com/ethereum/go-ethereum v1.12.0
```

Comment:
Where does 1.12.0 come from? Bumping to latest libocr should push this newer.

Reply:
Common update IIRC...

Reply:
@jmank88 is the current one ok?  If so, I would like to check this in to unblock any work on Starknet form updating common.